go.work

```diff
@@ -0,0 +1,10 @@
+go 1.20
```

Comment:
Sorry for the dumb question...what do go workspaces actually do for you?  Someone wanted to add a `go.work` file to our repo, but I didn't fully understand how it would help.  We have several modules and don't use a workspace, and things generally work fine.

Reply:
I think they are more useful in local development etc. when you have modules side-by-side rather than a module hierarchy like we have here, you *should* mainly be able to remove the `replace` directives but it doesn't seem (from my playing around with it as well) that we get that here because of the package/module structure

Reply:
e.g. to get the other modules in to actually use the toher modules in this repo all the way, had to do this: https://github.com/sunjayBhatia/go-control-plane/commit/4be6f62a029bd5e32eb259df831d2d3b9fb9e7cb

so I'm not sure how much the workspace is getting us

Reply:
Is the decision to ultimately remove the workspace? I'm fine with that